### PR TITLE
Plugins: Add descriptions to modules

### PR DESCRIPTION
**Pull request by kkartaltepe**
Does what it says on the tin.

### CODE REVIEW

**Comment by jp9000** [comment]
Functions should not have the opening brace on the same line as the function declaration.

Instead of:

```C
MODULE_EXPORT const char *obs_module_description(void) {
```
it should be:
```C
MODULE_EXPORT const char *obs_module_description(void)
{
```

**Comment by kkartaltepe** [comment]
Updated

File: plugins/obs-remote [hunk 1]
```diff
@@ -0,0 +1 @@
+../../obs-remote/
```
Comment (juvester) [inline plugins/obs-remote]: This symbolic link points to a folder that is outside the project root.

File: plugins/mac-capture/plugin-main.c [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("mac-capture", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "MacOS audio input/output and window/display capture";
```
Comment (RytoEX) [inline plugins/mac-capture/plugin-main.c]: I believe "macOS" _always_ has a lowercase "m", regardless of where it is in a sentence.

File: plugins/mac-syphon/plugin-main.c [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("syphon", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "Syphon based game capture for MacOS";
```
Comment (RytoEX) [inline plugins/mac-syphon/plugin-main.c]: "MacOS" should be "macOS".

File: plugins/obs-qsv11/obs-qsv11-plugin-main.c [hunk 4]
```diff
@@ -58,6 +58,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-qsv11", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "Intel QuickSyncVideo H.264 encoder support for Windows";
```
Comment (RytoEX) [inline plugins/obs-qsv11/obs-qsv11-plugin-main.c]: The correct style is "Intel® Quick Sync Video".

File: plugins/obs-remote (hunk 1 above)
Comment (RytoEX) [inline plugins/obs-remote]: This file seems to not be a currently included core/first-party plugin?  Was this added by mistake?

File: plugins/win-mf/mf-plugin.cpp [hunk 5]
```diff
@@ -30,6 +30,10 @@ extern "C" void obs_module_unload(void)
 }
 
 OBS_DECLARE_MODULE()
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "Windows Media Foundations H.264/AAC encoder";
```
Comment (RytoEX) [inline plugins/win-mf/mf-plugin.cpp]: I thought WMF was only used for audio.  Does AAC have to be designated as "H.264/AAC"?

File: plugins/obs-remote (hunk 1 above)
Comment (kkartaltepe) [reply plugins/obs-remote]: Lol this is what happens when i build in a dirty working dir. Thought I caught this, thanks.

File: plugins/obs-qsv11/obs-qsv11-plugin-main.c (hunk 4 above)
Comment (kkartaltepe) [reply plugins/obs-qsv11/obs-qsv11-plugin-main.c]: I know intel likes to use (R), but by the same token every company name and branded technology could have some kind of legal marker. Will split the Quick Sync Video part.

**Comment by kkartaltepe** [comment]
Thanks for the review.

**Pull request merged**